tests/data/simple_project/main.cf

```diff
@@ -1,3 +1,7 @@
 agent = std::AgentConfig(agentname=host.name, autostart=true)
 host = std::Host(name="localhost", os=std::linux, requires=agent)
 file = std::ConfigFile(host=host, path="/tmp/test", content="1234", requires=agent)
+val = std::get_env(name="my_unique_var",default_value="0")
```

Comment:
I would name the env var `add_one_resource` as this is not very descriptive naming 
And use 'true' as a value